component/cortex-tenant-ns-label.jsonnet

```diff
@@ -0,0 +1,125 @@
+local kap = import 'lib/kapitan.libjsonnet';
+local kube = import 'lib/kube.libjsonnet';
+
+local inv = kap.inventory();
+local params = inv.parameters.cortex_tenant_ns_label;
+
+local namespace_meta = {
+  metadata+: {
+    namespace: params.namespace,
```

Comment:
It's not necessary to explicitly specify `metadata.namespace` for resources which are deployed into the same namespace that's configured as the ArgoCD application's target namespace in app.jsonnet.

Reply:
I've tried removing this and it works for the deployment and the service, but it doesn't work for the serviceaccount.
```
Jsonnet error: failed to compile /cortex-tenant-ns-label/component/main.jsonnet:
 RUNTIME ERROR: field does not exist: namespace
	/usr/local/lib/python3.11/site-packages/commodore/lib/kube-libsonnet/kube.libsonnet:679:18-38	object <anonymous>
	/usr/local/lib/python3.11/site-packages/commodore/lib/kube-libsonnet/kube.libsonnet:(677:16)-(681:6)	thunk <array_element>
	/usr/local/lib/python3.11/site-packages/commodore/lib/kube-libsonnet/kube.libsonnet:(677:15)-(681:31)	object <anonymous>
	/cortex-tenant-ns-label/component/main.jsonnet:120:3-23	thunk <array_element>
	During manifestation	
```

I guess I'll leave things as they are.

Reply:
Yeah, some of the kube-libsonnet functions expect the namespace to be set. It doesn't break anything to specify the namespace explicitly, so it's fine either way.